schema.sql

```diff
@@ -23,6 +23,26 @@ CREATE TABLE species (
     name TEXT NOT NULL
 );
 
+CREATE TABLE vets (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name TEXT NOT NULL,
```

Comment:
- [ ] Kindly be noticed that it is better to make the name varchar(255) as it is much smaller than TEXT.